Tests/MongoSwiftSyncTests/MongoCollectionTests.swift

```diff
@@ -272,6 +272,28 @@ final class MongoCollectionTests: MongoSwiftTestCase {
         expect(deleteManyResult).to(beNil())
     }
 
+    func testRenamed() throws {
+        let encoder = BSONEncoder()
+        let client = try MongoClient.makeTestClient()
+        let db = client.db("testRenamedDB")
+        defer { try? db.drop() }
+        let coll = try db.createCollection("testRenamedColl")
+        let to = "newName"
+        let expectedWriteConcern: WriteConcern = try WriteConcern(journal: true, w: .number(1))
```

Comment:
this `: WriteConcern` type annotation isn't necessary. The compiler should be able to infer the type via the `WriteConcern` initializer.